src/kx122/kx122.hpp

```diff
@@ -50,9 +50,10 @@ namespace upm{
       @param bus I2C or SPI bus to use.
       @param addr I2C address of the sensor.
       @param chip_select Chip select pin for SPI.
+      @param bus_frequency Speed of the communication bus in Hz.
       @throws std::runtime_error on initialization failure.
       */
-      KX122(int bus, int addr, int chip_select);
+      KX122(int bus, int addr, int chip_select, int bus_frequency);
```

Comment:
A default argument for the bus frequency may minimize changes for developers already using the KX122:

```cpp
KX122(int bus, int addr, int chip_select, int bus_frequency = 10000);
```